frontend/crd/src/app/admin/task-edit-modal/task-edit-modal.component.ts

```diff
@@ -43,7 +47,11 @@ export class TaskEditModalComponent implements OnInit {
   }
 
   ngOnInit() { 
-    this.createForm();
+    this.eventService.getEvents().subscribe(events => {
```

Comment:
is this cached or is it making another network call? why not just pass in the event list through the modalData?

Reply:
this is the first time it's called so the task main page doesn't have the list